archie-utils/src/main/java/com/nedap/archie/json/OperatorLegacyFormatMixin.java

```diff
@@ -0,0 +1,16 @@
+package com.nedap.archie.json;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.nedap.archie.rules.Expression;
+import com.nedap.archie.rules.OperatorKind;
+
+public interface OperatorLegacyFormatMixin {
+    @JsonIgnore(false)
+    Expression getLeftOperand();
+    @JsonIgnore(false)
+    Expression getRightOperand();
```

Comment:
These two methods don't exist any more in the Operator class.

Reply:
I will remove them, and will add the backwards incompatibility to the change notes.